libexec/rc/rc.d/ldconfig

```diff
@@ -50,7 +49,7 @@ ldconfig_start()
 	if [ -x "${ldconfig_command}" ]; then
 		_LDC=$(/libexec/ld-elf.so.1 -v | sed -n -e '/^Default lib path /s///p' | tr : ' ')
```

Comment:
Another note: this is going to become more complicated when ld-elf.so.1 can be both c18n and not, especially if c18n is the default